sensor_msgs/Imu Imu
float32 wx
float32 wy
float32 wz